src/protocols/secure_channel/tests/TestCASESession.cpp

```diff
@@ -727,7 +727,7 @@ static CHIP_ERROR EncodeSigma1(MutableByteSpan & buf)
                                         ByteSpan(initiatorResumeMIC, Params::initiatorResumeMICLen)));
     }
 
-    if (Params::includeStructEnd)
+    if constexpr (Params::includeStructEnd)
```

Comment:
Constants should all be named `Params::kXxxxYyy`. The `k` prefix is missing.

Reply:
done